modules/unsupported/cql-json/src/main/java/org/geotools/filter/text/cqljson/model/Contains.java

```diff
@@ -0,0 +1,20 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2016, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+
+package org.geotools.filter.text.cqljson.model;
+
+public class Contains extends SpatialExpressions {}
```

Comment:
This used to be how enumerations were represented in Java (so they could be strongly typed).

Is making SpatialExpressions an enum worthwhile?

Reply:
I did this as a bit of future proofing and to match the CQLJson standard.  If you think this clutter things I can remove it.